SLHCUpgradeSimulations/Configuration/python/gemCustoms.py

```diff
@@ -51,4 +51,6 @@ def customise_gem_hlt(process):
     process.hltL2Muons.BWFilterParameters.EnableGEMMeasurement = cms.bool( True )
     return process
 
-
+def customiseME010EtaPart(process):
```

Comment:
Adding a new customization function is not acceptable.

If this setting should only be applied to the new ME0 geometry (and not the TP version), I propose putting it in GeometryExtended2023D6Reco_cff.py. (You should update the M2 reco dict entry appropriately.)